src/XCCDF/xccdf_session.c

```diff
@@ -890,6 +902,19 @@ int xccdf_session_load_oval(struct xccdf_session *session)
 			return 2;
 		}
 
+		if (session->export.thin_results) {
+			struct oval_results_model *res_model = oval_agent_get_results_model(tmp_sess);
+			struct oval_directives_model *dir_model = oval_results_model_get_directives_model(res_model);
+			// This is the worst function name in existence, despite its name,
+			// it's getting the oval_result_directives of the oval_directives_model.
+			// You would expect oval_directives_model_getresdirs at least, but no..
+			struct oval_result_directives *dir = oval_directives_model_get_defdirs(dir_model);
+			oval_result_directives_set_content(dir,  OVAL_RESULT_TRUE | OVAL_RESULT_FALSE |
```

Comment:
I removed the call because I think it's not necessary. Why do you think it's necessary?